### PR TITLE
Add announcements for week 11, 2021

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

File: announcements/2021/week11.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+[Loki v0.1.2-beta](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.2-beta/) adds the option to install loki-multi-tenant-proxy to ease multi-tenant authentication and authorization. Prior, [Loki v0.1.1-alpha2](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.1-alpha2/) upgrades to Loki v2.2.0.
+
+[Grafana v0.2.0](https://docs.giantswarm.io/changes/managed-apps/grafana-app/v0.2.0/) upgrades to upstream Grafana image v7.4.3.
+
+[Fluent Logshipping v0.6.4](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.4/) greaks up the stream names for Cloudwatch Output, to help prevent from hitting API limits.
+
+## Apps supported with best effort
+
+[Goldilocks v0.1.0]() is now available as an app supported with best effort. Some customers report significant cost savings from automatically setting requests and limits for clusters.
```
Comment (cokiengchiara) [inline announcements/2021/week11.md:15]: need link

File: announcements/2021/week11.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+[Cloudflared v0.0.5]() is now available as an app supported with best effort. This app allows you to launch Cloudflare Argo Tunnels and then route to services inside your cluster. Best for customers whose ingress options are constained and / or clusters run on-premises.
```
Comment (cokiengchiara) [inline announcements/2021/week11.md:17]: need link

**Review by yasn77 — APPROVED** [review]
Other than @cokiengchiara comment about missing links, it looks good 👍🏽

File: announcements/2021/week11.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Loki v0.1.2-beta](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.2-beta/) adds the option to install loki-multi-tenant-proxy to ease multi-tenant authentication and authorization. Prior, [Loki v0.1.1-alpha2](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.1-alpha2/) upgrades to Loki v2.2.0.
+
+[Grafana v0.2.0](https://docs.giantswarm.io/changes/managed-apps/grafana-app/v0.2.0/) upgrades to upstream Grafana image v7.4.3.
+
+[Fluent Logshipping v0.6.4](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.4/) greaks up the stream names for Cloudwatch Output, to help prevent from hitting API limits.
```
Comment (cokiengchiara) [inline announcements/2021/week11.md:11]:
```suggestion
[Fluent Logshipping v0.6.4](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.4/) breaks up the stream names for Cloudwatch Output, to help prevent from hitting API limits.
```

File: announcements/2021/week11.md (hunk 1 above)
Comment (cokiengchiara) [inline announcements/2021/week11.md:15]:
```suggestion
[Goldilocks v0.1.0]() is now available. Some customers report significant cost savings from automatically setting requests and limits for clusters.
```

File: announcements/2021/week11.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+[Grafana v0.2.0](https://docs.giantswarm.io/changes/managed-apps/grafana-app/v0.2.0/) upgrades to upstream Grafana image v7.4.3.
+
+[Fluent Logshipping v0.6.4](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.4/) breaks up the stream names for Cloudwatch Output, to help prevent from hitting API limits.
+
+## Apps supported with best effort
+
+[Goldilocks v0.1.0]() is now available. Some customers report significant cost savings from automatically setting requests and limits for clusters.
+
+[Cloudflared v0.0.5]() is now available as an app supported with best effort. This app allows you to launch Cloudflare Argo Tunnels and then route to services inside your cluster. Best for customers whose ingress options are constained and / or clusters run on-premises.
```
Comment (cokiengchiara) [inline announcements/2021/week11.md:17]:
```suggestion
[Cloudflared v0.0.5]() is now available. This app allows you to launch Cloudflare Argo Tunnels and then route to services inside your cluster. Best for customers whose ingress options are constained and / or clusters run on-premises.
```

**Review by cokiengchiara — APPROVED** [review]
- [ ]. Just need the links from docs.giantswarm.io

otherwise, lgtm. thanks @marians !

**Review by ced0ps — COMMENTED** [review]
Great work

**Pull request merged**